bff/src/Bff/Configuration/BffEndpointRouteBuilderExtensions.cs

```diff
@@ -58,15 +60,19 @@ public static void MapBffManagementLoginEndpoint(this IEndpointRouteBuilder endp
     /// Adds the silent login BFF management endpoints
     /// </summary>
     /// <param name="endpoints"></param>
+    [Obsolete("Silent login is now handled via login=create")]
```

Comment:
All of the Obsolete messages are inconsistent, and all would benefit from a longer explanation. 

```suggestion
    [Obsolete("The silent login endpoint will be removed in a future version. Silent login is now handled by passing the prompt=none parameter to the login endpoint.")]
```

Reply:
implemented